app/libs/notification_builder.rb

```diff
@@ -0,0 +1,37 @@
+class NotificationBuilder
+  attr_reader :content, :status, :notification_content, :user
+
+  def initialize(content: nil, user: nil)
+    raise ArgumentError('specify content') unless content
+    @content = content
+    find_user(user)
```

Comment:
mb `#set_user`

Reply:
![image](https://user-images.githubusercontent.com/32734554/36061430-ca4d5ac4-0e62-11e8-88ab-b6c00a910fbc.png)
Robocop doesn't like it